x-pack/plugin/core/src/main/resources/logs-default-mappings.json

```diff
@@ -0,0 +1,152 @@
+{
+  "template": {
+    "mappings": {
+      "_meta": {
```

Comment:
Could we add some metadata here? It would be nice to indicate in here that these are "stack" templates. The other thing that would be nice is to have the version of the stack that loaded it in here. 

What is the behaviour if the ES is upgraded, will these be overwritten?